crates/forge/tests/cli/ext_integration.rs

```diff
@@ -1,50 +1,57 @@
 use foundry_test_utils::util::ExtTester;
 
+// Actively maintained tests
+
+// <https://github.com/foundry-rs/forge-std>
 #[test]
 fn forge_std() {
-    ExtTester::new("foundry-rs", "forge-std", "2b59872eee0b8088ddcade39fe8c041e17bb79c0")
+    ExtTester::new("foundry-rs", "forge-std", "08d6af5d6c8a9a60e308b689cd19751876d321e0")
```

Comment:
Just for future reference, this commit is from this PR: https://github.com/foundry-rs/forge-std/pull/643

Reply:
yeah, maybe we could have additional reviewers for that forge-std PR, bump @mds1 :)